yarn-project/stdlib/src/versioning/versioning.ts

```diff
@@ -145,3 +149,28 @@ export function getVersioningResponseHandler(versions: Partial<ComponentsVersion
     return Promise.resolve();
   };
 }
+
+/**
+ * Returns package version.
+ * If .release-please-manifest.json could not be read or parsed returns empty string
+ */
+export function getPackageVersion(logger?: Logger): string {
+  try {
```

Comment:
@alexghr We will want to consolidate this with the work we are doing around versioning.